TodoListService/appsettings.json

```diff
@@ -1,16 +1,21 @@
 ﻿{
-    "Logging": {
-        "IncludeScopes": false,
-        "LogLevel": {
-            "Default": "Debug",
-            "System": "Information",
-            "Microsoft": "Information"
-        }
+  "AzureAd": {
+    "Instance": "https://login.microsoftonline.com/",
+    "Domain": "qualified.domain.name", // for instance contoso.onmicrosoft.com
```

Comment:
I think 'Domain' would cause some confusion. I would replace this to 'Tenant' and add a comment like:  // Tenant is the tenant ID (e.g. contoso.onmicrosoft.com, or 'common' for multi-tenant)